setup.py

```diff
@@ -24,20 +24,9 @@
     install_requires=[
         'forte~=0.2.0',
         "sortedcontainers==2.1.0",
-        "numpy>=1.16.6",
-        "jsonpickle==1.4",
-        "pyyaml==5.4",
-        "smart-open>=1.8.4",
-        "typed_astunparse==2.1.4",
-        "funcsigs==1.0.2",
         "mypy_extensions==0.4.3",
         "typed_ast>=1.4.3",
-        "jsonschema==3.0.2",
         "texar-pytorch",
-        'typing>=3.7.4;python_version<"3.5"',
-        "typing-inspect>=0.6.0",
-        'dataclasses~=0.7;python_version<"3.7"',
-        'importlib-resources==5.1.4;python_version<"3.7"',
         'dataclasses~=0.7;python_version<"3.7"',
         "fastapi==0.65.2",
         "uvicorn==0.14.0",
```

Comment:
I wonder why are these dependencies not removable? I don't think most of these dependencies are used in this repo